src/config.jl

```diff
@@ -211,19 +173,20 @@ Retrieves the current configuration value.
 Among the registered configuration files, fetches the value in order of priority (see `Base.lt(::ConfigFileOrder, path1, path2)`).
 If the key path does not exist in any of the configurations, returns `nothing`.
 """
-function get_config(manager::ConfigManager, key_path::String...)
-    is_static_setting(key_path...) &&
-        return access_nested_dict(load(manager).static_settings, key_path...)
-    for config in values(load(manager).watched_files)
-        return @something access_nested_dict(config, key_path...) continue
+function get_config(manager::ConfigManager, key_path::Symbol...)
```

Comment:
Should we use macros that expand to direct field access to make type inference results more precise?